### ISSUE TITLE
FR: Possibility for different areas to go to next/ previous page.

### ISSUE THREAD

**Issue by sandersantema** [koreader/koreader#4053]
Like you can do on cobalt. This might already be an option however I can't find it anywhere. Possibly due to searching for the wrong thing. Anyways thanks for the excellent software, it's a joy to use on my Kobo Aura 2!
![kobo-aura-hd-review-9](https://user-images.githubusercontent.com/21057514/42274996-02e76562-7f8f-11e8-868b-18e13a596be9.jpg)

**Comment by cramoisi** [comment]
it's nickel :)

**Comment by poire-z** [comment]
We should rename this project to `Kobalt`

**Comment by cramoisi** [comment]
let's play : 

k indle/Kobo
o 
b
a
l
t

**Comment by poire-z** [comment]
@sandersantema : you can copy and paste this to a new file (that won't be overwrriten by updates) named `defaults.persistent.lua` and play with the values:
https://github.com/koreader/koreader/blob/7d8ccda85241f03d3911d2210da870dffc72a67e/defaults.lua#L66-L79

**Comment by AlanSP1** [comment]
Here are mine settings:

DTAP_ZONE_MENU = {x = 1/8, y = 0, w = 3/4, h = 1/12}
DTAP_ZONE_CONFIG = {x = 0, y = 0, w = 0, h = 0}
DTAP_ZONE_MINIBAR = {x = 0.1, y = 0.97, w = 0.9, h = 0.03}
DTAP_ZONE_FORWARD = {x = 1/12, y = 0, w = 11/12, h = 1}
DTAP_ZONE_BACKWARD = {x = 0, y = 0, w = 1/12, h = 1}
DTAP_ZONE_BOOKMARK = {x = 7/8, y = 0, w = 1/8, h = 1/12}
DTAP_ZONE_FLIPPING = {x = 0, y = 0, w = 1/8, h = 1/12}
DDOUBLE_TAP_ZONE_NEXT_CHAPTER = {x = 6/8, y = 0, w = 2/8, h = 2/8}
DDOUBLE_TAP_ZONE_PREV_CHAPTER = {x = 0, y = 0, w = 2/8, h = 2/8}

Basically, I have small area to left edge for going back and almost all of the rest of screen is for going forward (except small parts in top for accessing menu, and lowest part for minibar (status bar).

You also see that you don't need to use fractions, you can use decimal numbers as well.

**Comment by AlanSP1** [comment]
Isn't it in koreader's root directory (not sure where it is in Android)?

**Comment by myelsukov** [comment]
Yes it is. I was confused because there were no defaults.lua file there. I just added a file ant it worked.
On a side note: built-in koreader file browser doesn't show files in the /koreader directory even though 'Show system files" option is on. It shows only directories. Is it expected?
I had to use other file browser to see the files in the /koreader directory

**Comment by Frenzie** [comment]
It currently only shows files it can open.
